orchestrator/api/community_skills.py

```
@@ -0,0 +1,303 @@
+
+    try:
+        async with httpx.AsyncClient(timeout=30, follow_redirects=True) as client:
+            resp = await client.get(skill_url)
+            if resp.status_code != 200:
+                raise HTTPException(
+                    status_code=502,
+                    detail=f"Failed to download skill: HTTP {resp.status_code}",
+                )
+            content = resp.text
+    except httpx.HTTPError as exc:
+        raise HTTPException(status_code=502, detail=f"Download error: {exc}")
```

Ruff (0.15.0)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)